### DISCUSSION TITLE
Add alternative login in the Web3Modal

### DISCUSSION THREAD

**Discussion by akugone** [WalletConnect/walletconnect-monorepo#2677] [Developer Support]
Hello
We are starting to work with a complementary login system that use a kind of KYC (anima.io), is it possible to add it directly in the webModal pop up as a proper login icon in "view all"  (is it on the code side or does the app have to register herself on your platform ?) or should have add the alternative connection over the Web3Modal.
In the second case where i have to implement over the Web3Modal, have you some usecase that i could study cause i don't really get how i could implement it as we use Web3Modal.

**Comment by xzilja** [comment]
As of now, best solution would be creating a wagmi connector around your flow, as this makes it very easy for users to add it in web3modal and in any other wagmi app in general.

**Reply by akugone** [reply]
Ok thanks i will try that
